client/src/reducers/genesetsUI.js

```diff
@@ -0,0 +1,55 @@
+/*
+Reducers for geneset UI-state.
+*/
+const GeneSetsUI = (
+  state = {
+    createGenesetModeActive: false,
+    isEditingGenesetName: false,
+    isAddingGenesToGeneset: false,
+  },
+  action
+) => {
+  switch (action.type) {
```

Comment:
please document the interface of each action, at least briefly (ie, if the action name is clear, at least add the parameters and their type).